flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontendParser.java

```diff
@@ -164,6 +164,14 @@ public class CliFrontendParser {
                     false,
                     "Defines whether to trigger this checkpoint as a full one.");
 
+    static final Option SAVEPOINT_DETACH_OPTION =
```

Comment:
1. How about just using 'detached' here aligned with DETACHED_OPTION ? It should not be conflict with other one because it will only be checked in savepoint and stop-with-savepoint command, right ? It will help us migrating in the future.
2. I saw this could also be used for manual checkpoint. Of course, we could support this in other ticket/pr.

Reply:
1. Good advice, I think using 'detached' option here can make it easily for users to understand, thanks for comment.
2. Yes, I have notice this point too, maybe we could finish this ticket first ~